src/tds/codec/column_data.rs

```diff
@@ -480,8 +480,14 @@ impl<'a> ColumnData<'a> {
             src.read_i32_le().await?; // days
             src.read_u32_le().await?; // second fractions
 
-            let text_len = src.read_u32_le().await? as usize / 2;
-            let text = read_varchar(src, text_len).await?;
+            let len = src.read_u32_le().await? as usize / 2;
```

Comment:
So this one is special because the length is u32, right?

Reply:
This is the very special `NTEXT` type, a deprecated thing we must live with forever. The length is in bytes, which is different than `US_VARCHAR` or `B_VARCHAR`.